sdk/face/ai-vision-face-rest/test/public/faceSessionClient.spec.ts

```diff
@@ -0,0 +1,562 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { createRecorder, createClient } from "./utils/recordedClient.js";
+import { assert, beforeEach, afterEach, it, describe } from "vitest";
+import {
+  Recorder,
+  // isLiveMode,
+  isPlaybackMode,
+} from "@azure-tools/test-recorder";
+
+import { FaceClient, LivenessResponseBodyOutput, isUnexpected } from "../../src/index.js";
```

Comment:
Do we have a change to verify the helper `getLongRunningPoller` considering your LRO is non-standard?

Reply:
You mean the API returning 200 and 202, right? Let me add a test for it.

Reply:
yes, i notice you add samples in README and pls ensure they work!